packages/axios-extension/src/auth/connection.ts

```diff
@@ -0,0 +1,180 @@
+import { AxiosResponse, AxiosRequestConfig, AxiosError } from 'axios';
+import { ServiceProvider } from '../base/service-provider';
+import { ConnectionError } from './error';
+import detectContentType from 'detect-content-type';
+
+export enum CSRF {
+    requestHeaderName = 'X-Csrf-Token',
+    requestHeaderValue = 'Fetch',
+    responseHeaderName = 'x-csrf-token'
```

Comment:
Enum members are conventionally named starting with an uppercase letter. Should we change to follow that?

Reply:
Yes, please.